ios/Video/RCTVideo.swift

```diff
@@ -448,13 +448,16 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
         #endif
 
         if source.drm != nil || _localSourceEncryptionKeyScheme != nil {
-            _resouceLoaderDelegate = RCTResourceLoaderDelegate(
+            if _drmManager == nil {
+                _drmManager = DRMManager()
+            }
+
+            _drmManager?.createContentKeyRequest(
                 asset: asset,
-                drm: source.drm,
-                localSourceEncryptionKeyScheme: _localSourceEncryptionKeyScheme,
```

Comment:
localSourceEncryptionKeyScheme should be moved inside drm prop no ?

Reply:
This comment is not mandatory to merge the PR !

Reply:
I moved it but without braking change - deprecated old one
Forgot to update docs - I will do it soon